### DISCUSSION TITLE
2024 Q2 POAP Eligibility List and Contributor Incentives

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#644] [Announcements]
## POAP Eligibility List POAP 领取资格名单

We are excited to announce the eligibility list of contributors who are qualified to claim Proof of Attendance Protocol (POAP) tokens for 2024 Q2. We deeply appreciate your support and highly value the insights and expertise you bring to our community.

以下为2024 Q2季度有资格领取 POAP 的贡献者名单。非常感谢名单中的各位对本项目的支持。

The following Crowdin contributors are eligible:

以下的Crowdin贡献者有资格领取POAP：

* [effectchen](https://crowdin.com/profile/effectchen)
* [conflux.pei](https://crowdin.com/profile/conflux.pei)
* [w33d](https://crowdin.com/profile/w33d)
* [PingJing](https://crowdin.com/profile/PingJing) (distinguished)
* [Axxx036](https://crowdin.com/profile/Axxx036)
* [devblue4](https://crowdin.com/profile/devblue4)

The following GitHub contributors are eligible:

以下的GitHub贡献者有资格领取POAP：

* [Pana](https://github.com/Pana)
* [darwintree](https://github.com/darwintree)
* [0xn1c0](https://github.com/0xn1c0)
* [ClockRide](https://github.com/ClockRide)
* [devblue4](https://github.com/devblue4)
* [jackleeio](https://github.com/jackleeio)(distinguished)
* [lululu00](https://github.com/lululu00)
* [shengchenshengchen](https://github.com/shengchenshengchen)
* [iosh](https://github.com/iosh)

The eligibility list is curated in accordance with our [CONTRIBUTING guideline](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING). If you believe your contribution has been overlooked, please do not hesitate to contact us.

白名单是根据[CONTRIBUTING指导原则](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING)所挑选的。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。

**The mint deadline is block number 245585539 (Aug 2nd 2024 15:28:00 UTC+08:00).**

**铸造的截止时限为区块号 245585539（2024年08月02日 15:28:00 UTC+08:00）。**

Please be aware that the POAP issuance rules are subject to review and potential modifications every 3 months. This ensures that our guidelines remain pertinent and effective in encouraging and rewarding valuable contributions.

请注意，POAP的发行规则每3个月会进行审查和可能的修改，以确保我们的指导规则仍然可以有效地鼓励和奖励社区的贡献。

## Contributor Incentives 贡献者激励

In addition to POAP, further financial incentives will be distributed to the documnetation contributors, including both translation contributors and documentation content contributors. These incentives are **directly** related to the quality of the contributor's work.

除了POAP外，文档贡献者（包括翻译贡献者与文档内容贡献者）还能获得更多经济激励。这些激励是与贡献者的工作质量所**直接**相关的。

At present, we offer corresponding incentives for Chinese translators. Translator incentives are calculated based on the volume of translations and whether the translation is accepted. We will airdrop rewards to translators who have provided their addresses. For the method of providing addresses, please refer to [Conflux Technical Documentation Translation Plan (2024.03.06) ](https://forum.conflux.fun/t/conflux-2024-03-06/20286). It should be noted that this airdrop will also provide incentives for translators before January 2024. 

目前我们为中文翻译者提供相应激励，翻译者激励计算与翻译量与翻译是否被接受相关。我们将为已提供了地址的翻译者空投激励。地址提供方式请参考[Conflux 技术文档翻译计划（2024.03.06）](https://forum.conflux.fun/t/conflux-2024-03-06/20286)。

For incentives for documentation content contributors, please refer to [Conflux Doc Contributions DB (Apr 2024 ~)](https://confluxglobal.notion.site/9a878268683d492dae043e4878f724a8?v=fcf09b97de6b47eab60cd00a8946f3e9&pvs=4), [Conflux Doc Contributions DB (May 2024 ~June 13th 2024)](https://confluxglobal.notion.site/f1f6955cee6244b3a49eeffe10021399?v=f56533480f4e4d7483f631c673c10ae4&pvs=4), [Conflux Doc Contributions DB (June 14th 2024～June 30th 2024)](https://confluxglobal.notion.site/cde53a26b59f432484429db37e92705b?v=b0dd6ef33f7a4d51895250dbd6cc29a6&pvs=4) and their incentives are calculated according to the [2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md). If you have relevant contributions but have been omitted from the list, please feel free to contact us. In addition, we will collect the (temporary) addresses provided by the relevant contributors for airdrop redemption in this discussion.

文档内容贡献者的激励请参考[Conflux Doc Contributions DB (Apr 2024 ~)](https://confluxglobal.notion.site/9a878268683d492dae043e4878f724a8?v=fcf09b97de6b47eab60cd00a8946f3e9&pvs=4)，[Conflux Doc Contributions DB (June 14th 2024～June 30th 2024)](https://confluxglobal.notion.site/f1f6955cee6244b3a49eeffe10021399?v=f56533480f4e4d7483f631c673c10ae4&pvs=4)和[Conflux Doc Contributions DB (June 14th 2024～June 30th 2024)](https://confluxglobal.notion.site/cde53a26b59f432484429db37e92705b?v=b0dd6ef33f7a4d51895250dbd6cc29a6&pvs=4)。其激励根据[2024Q2 reward guidelines](https://github.com/Conflux-Chain/conflux-documentation/blob/2024Q1-reward/docs/general/CONTRIBUTING/reward-guidelines.md)计算。如果您有对应贡献但被遗漏在了名单外，欢迎联系我们。此外我们将在本discussion中收集相关贡献者提供的领取空投的（临时）地址。

>Please note that specific details may be adjusted in each new quarter, for details, please refer to [regular review of reward guideline](https://github.com/Conflux-Chain/conflux-documentation/pull/508).
>需要说明的是，在每个新的季度中，特定细则可能会调整，详情请参考[regular review of reward guideline](https://github.com/Conflux-Chain/conflux-documentation/pull/508)。

Thank you for your assistance in enhancing our project. If you have any queries or require assistance, please feel free to create an issue in our Github repository.

感谢各位持续帮助我们改进项目。如果你还有任何疑问或需要帮助，请随时在我们的Github仓库中创建issue。

### How to Claim the Contributor POAP 如何领取贡献者 POAP

1. Visit the [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app/). 访问 [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app/)。
2. Choose the space(s) in which you want to mint the POAP. You can choose Core Space, eSpace, or both (Dual Space). It is recommended to select core space or dual space because FC reward is only claimable on Conflux Core Space. 选择你想铸造到的 Space，你可以铸造到Core Space，eSpace 或者是同时铸造到两个 Space。我们推荐你铸造到 Core Space 或同时铸造到两个 Space 因为在 Core 空间能够领取 NFT 相关联的 FC 奖励。
3. Input your wallet address OR connect your wallet to automatically fill in the address. 输入你的钱包地址或者连接你的钱包以自动输入。
4. Choose the appropriate authorization method (GitHub or Crowdin). 选择合适的授权方式（Github 或 Crowdin）。
5. Mint your POAP! 现在即可铸造你的POAP。

<img width="1270" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/e649ca37-1acc-47c1-894c-760456110e96">

>Contact us if you are notified to have no permission.
>如果你被提示无权限进行铸造，请联系我们。

### How to Get POAP-Attached FC Rewards 如何领取 POAP 关联的 FC 奖励

We have set up a [Contributor Rewards Pool](https://confluxscan.io/address/cfx:achm6kb92by13rpvwd04zscn1127zuaseu25usm7sc) to reward holders of Proof of Attendance Protocols (POAPs) with FansCoin. This is the POAP-attached incentive and is calculated independently from other incentives.

我们设立了一个[贡献者奖励池](https://confluxscan.io/address/cfx:achm6kb92by13rpvwd04zscn1127zuaseu25usm7sc)，以用FansCoin奖励文档POAP的持有者。这一激励为POAP持有者提供附加的奖励，与其他激励独立结算。

1. Make sure your core account owning POAP is connected to the [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app/). Your account is supposed to own CFX to pay for gas and storage collateral. 确认你已经将 Core 空间账户连接到了 [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app/)。你的账户上需要有一定的CFX用于支付gas与存储抵押。
2. Click "Core Manage" on the left, you will see your POAP listed. 点击左侧的 "Core Manage"，你会看到 Core 账户持有的 POAP。
3. Select "Token ID" in the "Claim FC Reward" card, it will show how much you could claim. 在"Claim FC Reward"卡片中选择"Token ID"，将会显示你可以领取多少FC。
4. Click the "Claim xx FC" button and your wallet will be invoked. You can claim FC for each POAP you own (the FC amount available for claim may vary).点击"Claim xx FC"按钮，你的钱包将会被唤起。对于你持有的每个POAP，你都可以领取一次（不同的POAP可领取的数量会有所区别）。

<img width="1285" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/d73f17b4-76e7-44b0-9b7c-f47edd88a45e">

>For the POAP issued in the previous quarters, you can visit "Set Core Owner" card in "eSpace Manage" to set the Core Space owner.
>对于之前季度发放的POAP，你可以访问"eSpace Manage"页面的"Set Core Owner"卡片来设置Core Space的所有者。

<img width="1243" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/d298d981-b4c0-41fc-8fd9-67e05adcdfa0">

**Comment by darwintree** [comment]
@Pana @0xn1c0 @ClockRide @devblue4 @jackleeio @lululu00 @shengchenshengchen @iosh 
Thank you for your contribution in the past quarter! And you are eligible to claim the contributor POAP for the last quarter

And for @shengchenshengchen @iosh, Please leave your (temporary) core space address here for FC airdrop.

**Comment by darwintree** [comment]
@jcbm123143 Your airdrop for 2024 Q1 is still reserved. Please let us know your address on Conflux core space for airdrop

**Reply by unijaychie** [reply]
> @jcbm123143 Your airdrop for 2024 Q1 is still reserved. Please let us know your address on Conflux core space for airdrop

0x192513387b1bc31597ebf9f858e19bcb410ead6b

here my wallet address sorry I forget to claim my rewards

**Comment by jackleeio** [comment]
[Conflux Doc Contributions DB (June 14th 2024～June 30th 2024)](https://confluxglobal.notion.site/cde53a26b59f432484429db37e92705b?v=b0dd6ef33f7a4d51895250dbd6cc29a6) Could it be possible that this part of the incentive was missed? I haven’t received it.

**Reply by darwintree** [reply]
Sorry for that the incentive for May is missed. Already notified relating people to handle it

**Reply by jackleeio** [reply]
It hasn’t been received yet @darwintree

**Reply by darwintree** [reply]
> It hasn’t been received yet @darwintree

Asked and was told it will be distributed together with July incentive in early August
